contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -20,7 +20,7 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
   using FunctionsResponse for FunctionsResponse.Commitment;
   using FunctionsResponse for FunctionsResponse.FulfillResult;
 
-  uint32 private constant REASONABLE_GAS_PRICE_CEILING = 1_000_000;
+  uint56 private constant REASONABLE_GAS_PRICE_CEILING = 1_000_000_000_000_000; // 1 million wei
```

Comment:
comment should say "1 million gwei"

Reply:
Actually, I think this is correct.  We switched from gwei to wei for the units here. Right @justinkaseman ?

Reply:
This number represents 1 quadrillion wei or 1 million gwei so the comment is incorrect.

Reply:
@bolekk  is correct, this is gwei